File: tasks/implementations/node/tasks/sort_integers.js
const fs = require('fs');
module.exports = function(size, fixturesRoot) {
  const path = `${fixturesRoot}/generated/sort/${size}.txt`;
  const values = fs.readFileSync(path, 'utf8').trim().split(/\s+/).map(Number).sort((a, b) => a - b);
  const total = values.slice(0, 100).reduce((acc, item) => acc + item, 0);
  console.log(total);
};

File: tasks/implementations/python2/tasks/sort_integers.py
from tasks.compat import emit, join_path, read_text


def run(size, fixtures_root):
    path = join_path(fixtures_root, 'generated', 'sort', '%s.txt' % size)
    values = [int(x) for x in read_text(path).split()]
    values.sort()
    emit(sum(values[:100]))
